### DISCUSSION TITLE
Code Reviewer Group

### DISCUSSION THREAD

**Discussion by lordfolken** [XCSoar/XCSoar#1121] [General]
Hi, 

I'd like to create a Code Reviewer group here on Github, that is added to PRs by default.

The idea is to the get code reviewed by people that know what the standards are, before @MaxKellermann needs to have a look at it.

I'd also like to add https://pre-commit.org as an additional check on PRs so that at least some basic code syle is upkept. 
https://xcsoar.readthedocs.io/en/latest/policy.html#code-style

Any volunteers?

**Comment by ygaillard** [comment]
I would like as far there is no time pressure.

**Comment by hjr** [comment]
I like the pre-commit idea, let robots work at first place.
I would also volunteer reviewing .. as time permits.

**Comment by lordfolken** [comment]
Thix to those volunteering. I have added a codeowners file here:

https://github.com/XCSoar/XCSoar/pull/1156

As described here: https://github.blog/2017-07-06-introducing-code-owners/

**Comment by lordfolken** [comment]
So the first PR with the code reviewers group is up.
